docs/dictionaries/plWordListReadme.txt

```diff
@@ -0,0 +1,4 @@
+Polish wordlist by: rjolina
```

Comment:
This information is incorrect. Please, replace it with the original readme from the `.zip` file, excluding the encoding and win 95 stuff.

As for the license, nothing is explicitly stated by the author, so I suppose, we could consider the word list is compiled from public domain sources. But it is definitely not GPL 2.0.